libraries/entities-renderer/src/RenderableImageEntityItem.cpp

```diff
@@ -59,10 +59,22 @@ void ImageEntityRenderer::doRenderUpdateAsynchronousTyped(const TypedEntityPoint
     _alpha = entity->getAlpha();
     _pulseProperties = entity->getPulseProperties();
 
+    bool nextTextureLoaded = _texture && (_texture->isLoaded() || _texture->isFailed());
     if (!_textureIsLoaded) {
         emit requestRenderUpdate();
+        if (nextTextureLoaded) {
+            float width = _texture->getWidth();
+            float height = _texture->getHeight();
+            glm::vec3 naturalDimensions = glm::vec3(1.0f, 1.0f, 0.01f);
+            if (width < height) {
+                naturalDimensions.x = width / height;
+            } else {
+                naturalDimensions.y = height / width;
+            }
+            entity->setNaturalDimension(naturalDimensions);
```

Comment:
this is kinda hacky.  models handle this a little cleaner because the renderable actually IS the game object, which is itself a hack.  instead of messing with all the image entity code and changing it around, we can sneakily set the dimensions on the game object even though it’s const by using a mutable

Reply:
done!